### PR TITLE
fixed the issue with ambigious panic

### PR DESCRIPTION
**Pull request by BNMetrics**
#493

### CODE REVIEW

**Comment by BNMetrics** [comment]
@peterebden. I actually thought of a few options as well, including the one you suggestions. But wouldn't it be a bit ambigious. I think the `cannot_run` can also apply to any test rules, because some tests actually fail if you use `plz run` instead of `plz test`..
Let me know your thoughts!

**Comment by peterebden** [comment]
Yeah it is sometimes useful to run tests with `plz run` although as you say it only works if they don't have/use any `data`.

Still don't think having an extra attribute is necessary though, I think `run_step` should just check the outputs itself.

**Comment by BNMetrics** [comment]
Okay, If that's the case, let's make it simpler by checking if it's got any `data`! :)

File: src/build/command_replacements.go [hunk 1]
```diff
@@ -192,7 +192,7 @@ func replaceSequenceLabel(state *core.BuildState, target *core.BuildTarget, labe
 }
 
 func checkAndReplaceSequence(state *core.BuildState, target, dep *core.BuildTarget, in string, runnable, multiple, dir, outPrefix, hash, test, allOutputs, tool bool) string {
-	if allOutputs && !multiple && len(dep.Outputs()) != 1 {
+	if allOutputs && !multiple && len(dep.Outputs()) > 1 {
```
Comment (peterebden) [inline src/build/command_replacements.go]: I think it should still panic for when there are no outputs; might be nice to special-case that though so the message is more explicit.

Comment (BNMetrics) [reply src/build/command_replacements.go]: There is another case checking `runnable` and `len(dep.Outputs()) == 0` as a special case later actually! :)

**Pull request merged**